### PR TITLE
Fix: Autocomplete component overlapping sidebar

### LINKED ISSUE #21781: 🐛 Bug Report: EntityOwnerPicker is over the navigation bar
### 📜 Description

When you are in the backstage catalog and you open the `EntityOwnerPicker` menu and then without closing it you choose an item from the navigation bar, the EntityOwnerPicker items are above the navigation bar.

### 👍 Expected behavior

The `EntityOwnerPicker` must either be closed or below the navigation bar.

### 👎 Actual Behavior with Screenshots

The `EntityOwnerPicker` is above the navigation bar.

image

image

### 👟 Reproduction steps

1 - Open EntityOwnerPicker
2 - Without closing it go to the navigation bar

### 📃 Provide the context for the Bug.

`EntityOwnerPicker` is above the navigation bar

### 🖥️ Your Environment

Not really environment specific

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/backstage/backstage/blob/master/CODE_OF_CONDUCT.md)

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by JeevaRamanathan**
## Hey, I just made a Pull Request!
Fixes #21781 

Screenshot:
image

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info]
- [x] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by JeevaRamanathan** [comment]
Hi @freben,  `plugins/proxy-backend/src/service/router.credentials.test.ts` the pipeline fails here but that backend part was untouched...your inputs on this please?

**Comment by jeem5110** [comment]
Hi @freben @Rugvip, could you please have a look into this PR and add your suggestions

File: .changeset/pretty-spoons-knock.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-scaffolder-react': minor
+'@backstage/plugin-catalog-react': minor
```
Comment (vinzscam) [inline .changeset/pretty-spoons-knock.md:3]:
```suggestion
'@backstage/plugin-scaffolder-react': patch
'@backstage/plugin-catalog-react': patch
```

**Review by vinzscam — CHANGES_REQUESTED** [review]

**Comment by JeevaRamanathan** [comment]
Hi [vinzscam](https://github.com/vinzscam), i have modified the requested changes

**Review by freben — APPROVED** [review]
alright

**Comment by freben** [comment]
@vinzscam You ok too?

**Review by vinzscam — APPROVED** [review]

**Pull request merged**